### PR TITLE
Package Refactor

### PR DESCRIPTION
**Pull request by mitchellsayer**
Relates to https://github.com/viaduct-ai/backlog/issues/242

This pr removes all code unnecessary to the functionality of HybridContentsManager. The root directories have been renamed and the dependencies have all been updated to reflect the cut down code. The testing suite has also been updated. To run tests, run the command `tox`.

### CODE REVIEW

File: hybrid_content_manager/ipycompat.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+"""Utilities for managing compat between notebook versions."""
+
+#  if notebook.version_info[0] >= 6:  # noqa
+#  raise ImportError("Jupyter Notebook versions 6 and up are not supported.")
```
Comment (devstein) [inline hybrid_content_manager/ipycompat.py]: Delete these comments now that we removed the `pgcontents` code that was incompatible with `notebook >= 6.0.0`

File: setup.py [hunk 2]
```diff
@@ -22,13 +21,13 @@ def main():
...
         long_description=long_description,
-        author="Scott Sanderson",
-        author_email="ssanderson@quantopian.com",
-        packages=find_packages(include='pgcontents.*'),
+        author="Viaduct AI",
+        author_email="engineering@viaduct.ai",
+        packages=find_packages(include='hybrid_content_manager.*'),
         license='Apache 2.0',
         include_package_data=True,
         zip_safe=False,
```
Comment (devstein) [inline setup.py]: Check to see if we still need this https://setuptools.readthedocs.io/en/latest/setuptools.html#setting-the-zip-safe-flag

File: setup.py [hunk 3]
```diff
@@ -22,13 +21,13 @@ def main():
...
-        name='pgcontents',
+        name='hybrid-content-manager',
         version='0.6',
-        description="A Postgres-backed ContentsManager for IPython/Jupyter.",
+        description="Hybrid Content Manager",
         long_description=long_description,
-        author="Scott Sanderson",
-        author_email="ssanderson@quantopian.com",
-        packages=find_packages(include='pgcontents.*'),
+        author="Viaduct AI",
```
Comment (devstein) [inline setup.py]: viaduct.ai

File: setup.py [hunk 4]
```diff
@@ -40,18 +39,15 @@ def main():
             'Natural Language :: English',
             'Operating System :: OS Independent',
             'Programming Language :: Python :: 2.7',
-            'Programming Language :: Python :: 3.5',
             'Programming Language :: Python :: 3.6',
+            'Programming Language :: Python :: 3.7',
             'Programming Language :: Python',
             'Topic :: Database',
```
Comment (devstein) [inline setup.py]: Remove `Topic :: Database` and check if we should update `Development Status :: 4 - Beta`

Comment (devstein) [reply setup.py]: Also update `url="..."`

File: .travis.yml [hunk 5]
```diff
@@ -3,27 +3,21 @@ sudo: false
 
 python:
```
Comment (devstein) [inline .travis.yml]: Delete the `.travis.yml` file. We will switch to Github actions

**Comment by devstein** [comment]
Add instructions for testing to the `README`

File: setup.py (hunk 4 above)
Comment (mitchellsayer) [reply setup.py]: After looking at all of the options for `Development Status` I think we should leave it as it is.

**Pull request merged**